cameraServer.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+import json
+import time
+import sys
+import numpy as np
+import cv2
+import math
+
+#Magic Numbers
+lowerGreen = (38, 125, 100) #Our Robot's Camera
+higherGreen = (52, 255, 180)
+sample_lowerGreen = (30, 177, 80) #FRC sample images
```

Comment:
What variable naming convention are you using, [Camel Case](https://en.wikipedia.org/wiki/Camel_case) for two lines then [Snake Case](https://en.wikipedia.org/wiki/Snake_case) for two lines :laughing:, decide on one